Controller/Process/Result.php

```diff
@@ -107,7 +107,10 @@ public function execute()
                 $session->getQuote()->setIsActive(false)->save();
                 $this->_redirect('checkout/onepage/success', ['_query' => ['utm_nooverride' => '1']]);
             } else {
-                $this->_cancel($response);
+                $this->_adyenLogger->addAdyenResult(
+                    __('Payment for order %1 was unsuccessful, it will be cancelled when the OFFER_CLOSED notification has been processed.', $this->_order->getIncrementId())
```

Comment:
```suggestion
                    sprintf(
                        'Payment for order %d was unsuccessful, ' .
                        'it will be cancelled when the OFFER_CLOSED notification has been processed.',
                        $this->_order->getIncrementId()
                    )
```

Reply:
@msilvagarcia good point. I'll change the integer specifier %d for a string %s since order numbers can have leading zeros.